dist/stat_data.py

```diff
@@ -260,6 +260,7 @@ def __init__(self, name, desc, flags=''):
     CacheStat('cache_hazard_walks', 'hazard pointer check entries walked'),
     CacheStat('cache_inmem_split', 'in-memory page splits'),
     CacheStat('cache_inmem_splittable', 'in-memory page passed criteria to be split'),
+    CacheStat('cache_lookaside_cursor', 'sleep for cache overflow table cursor'),
```

Comment:
I agree with Michael, we should count the total time we wait here. It will be awesome if we can rename "cache_lookaside_cursor" to something accordingly, "cache_lookaside_cursor_acquire_time" maybe.